### PR TITLE
Release 12.1.1

### PR DESCRIPTION
**Pull request by r0mant**

### CODE REVIEW

**Review by russjones — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
 
+## 12.1.1
+
+This release of Teleport contains multiple improvements and bug fixes.
+
+* Fixed issue with Access Management's connection tester not working with per-session MFA. [#22918](https://github.com/gravitational/teleport/pull/22918), [#22943](https://github.com/gravitational/teleport/pull/22943)
+* Fixed Kubernetes access panic when using moderated sessions. [#22930](https://github.com/gravitational/teleport/pull/22930)
+* Fixed `tsh db config` reporting incorrect port in TLS routing mode. [#22889](https://github.com/gravitational/teleport/pull/22889)
+* Fixed issue with Teleport always performing OS group check even without auto user provisioning enabled. [#22805](https://github.com/gravitational/teleport/pull/22805)
+* Fixed issue with desktop access crashing on systems that consume many file descriptors. [#22798](https://github.com/gravitational/teleport/pull/22798)
```
Comment (zmb3) [inline CHANGELOG.md:11]: This PR fixes two things actually. In addition to the crash mentioned here, it also fixes unexpected session disconnects.

**Review by zmb3 — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 12.1.1
+
+This release of Teleport contains multiple improvements and bug fixes.
+
+* Fixed issue with Access Management's connection tester not working with per-session MFA. [#22918](https://github.com/gravitational/teleport/pull/22918), [#22943](https://github.com/gravitational/teleport/pull/22943)
+* Fixed Kubernetes access panic when using moderated sessions. [#22930](https://github.com/gravitational/teleport/pull/22930)
```
Comment (ptgott) [inline CHANGELOG.md:8]:
```suggestion
* Fixed Kubernetes Service panic when using Moderated Sessions. [#22930](https://github.com/gravitational/teleport/pull/22930)
```

File: CHANGELOG.md (hunk 1 above)
Comment (ptgott) [inline CHANGELOG.md:11]:
```suggestion
* Fixed issue with the Windows Desktop Service crashing on systems that consume many file descriptors. [#22798](https://github.com/gravitational/teleport/pull/22798)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+* Fixed issue with desktop access crashing on systems that consume many file descriptors. [#22798](https://github.com/gravitational/teleport/pull/22798)
+* Fixed issue with `teleport start --bootstrap` command failing on unexpected resource. [#22721](https://github.com/gravitational/teleport/pull/22721)
+* Fixed issue with install script not refreshing repository metadata before installing new version. [#22585](https://github.com/gravitational/teleport/pull/22585)
+* Added ability to export database CA in DER format via `tctl auth export`. [#22896](https://github.com/gravitational/teleport/pull/22896)
+* Reduced log spam from proxy multiplexer. [#22802](https://github.com/gravitational/teleport/pull/22802)
+* Updated EC2 auto-discovery install script to use enterprise binaries for enterprise clusters. [#22769](https://github.com/gravitational/teleport/pull/22769)
+* Upgraded Go to `v1.19.7`. [#22725](https://github.com/gravitational/teleport/pull/22725)
+* Improved idle connections handling. [#22908](https://github.com/gravitational/teleport/pull/22908), [#22893](https://github.com/gravitational/teleport/pull/22893)
```
Comment (ptgott) [inline CHANGELOG.md:18]:
```suggestion
* Improved idle connection handling. [#22908](https://github.com/gravitational/teleport/pull/22908), [#22893](https://github.com/gravitational/teleport/pull/22893)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+* Improved Kubernetes service labels validation upon startup. [#22777](https://github.com/gravitational/teleport/pull/22777)
```
Comment (ptgott) [inline CHANGELOG.md:19]:
```suggestion
* Improved Kubernetes Service label validation upon startup. [#22777](https://github.com/gravitational/teleport/pull/22777)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+* Improved `tsh login` error reporting when proxy is not available. [#22763](https://github.com/gravitational/teleport/pull/22763)
```
Comment (ptgott) [inline CHANGELOG.md:20]:
```suggestion
* Improved `tsh login` error reporting when the Proxy Service is not available. [#22763](https://github.com/gravitational/teleport/pull/22763)
```

**Review by ptgott — APPROVED** [review]

**Pull request merged**